File: tarea semana 3/playlist.cpp
#include <iostream>
using namespace std;
class playlist{
    public:
        string nombre;
        string estado;
        string indice;
        string canciones[100]={};
    public:
        playlist(string, string);
        void agrega_cancion();
        void elimina_cancion();
        void mostrar_canciones();
        void reproducir();
        void seleccionar_cancion();
        void pausar();
        void detener();
        void siguiente_cancion();
        void cancion_anterior();
        void aleatorio();
        void estado_playlist();
        void mostrar_canrep();

};
class cancion{
    public:
        int indice;
        string nombre;
    public:
        cancion(int,string);
};


playlist::playlist(string _nombre, string _estado){
    nombre = _nombre;
    estado= _estado;
}

int main(){
    int b;
    string v;
    cout<<"ingrese un id ", cin>>b;
    cout<<"ingrese un nombre ", cin>>v;
    cancion c1= cancion(b,v);

}

File: tarea semana 3/playlist.py
class musica:
    def __init__(self,nom, artista, genero):
        self.nom=nom
        self.artista=artista
        self.genero=genero
        self.estado= False
class playlist:
    def __init__(self):
        self.canciones=[]
    def agregar(self, can):
        self.cancione.append(can)
    def eliminar(self,can):
        self.canciones.remove(can)
    def mostrar_canciones(self):
        for k in range(len(self.canciones)):
            print(f"{k}. {self.canciones[k]}")
    def reproducir(self,x):
        self.canciones[x].estado=True
        if self.canciones[x] == True:
            print("Se esta reproduciendo")
    def pausar(self,x):
        if self.canciones[x].estado== True:
            self.canciones.estado= False
        else:
            print("la cancion ya esta pausada")
    def siguiente(self, ):
        self.canciones